shell/platform/darwin/ios/framework/Source/FlutterEngine.mm

```diff
@@ -533,7 +533,9 @@ - (void)sendOnChannel:(NSString*)channel message:(NSData*)message {
 - (void)sendOnChannel:(NSString*)channel
               message:(NSData*)message
           binaryReply:(FlutterBinaryReply)callback {
-  NSAssert(channel, @"The channel must not be null");
+  NSAssert(channel, @"The channel must not be nil");
+  NSAssert(_shell && _shell->IsSetup(),
+           @"Sending a message before the FlutterEngine has been run.");
```

Comment:
Can we tell them how to run it?

Reply:
I was careful to use the word "run" to match the verb in our run methods.  It's a bit complicated, should I say something like "See run* methods"?

Reply:
Eh it's okay, it would make sense to list it if there was maybe only one way to run it.  What you have is fine.